tests/v0_tests/test_get_settings.py

```diff
@@ -27,7 +27,6 @@ def test_default_settings(self):
         self.assertIn("number_of_replicas", index_settings)
         self.assertTrue(fields.issubset(set(index_settings['index_defaults'])))
 
-    @mark.ignore_during_cloud_tests
```

Comment:
I am not sure, isn't purpose of the test in getting index settings and verifying that they match? That's possible without mocking http there as it is not checked for open source as well and get_indexes is being tested there.
Or do you mean that we could change index settings and not be aware that it would fail this test?

Reply:
You are correct